root/layout/components/sidebar/ReleaseSidebar.js

```diff
@@ -77,7 +82,8 @@ const ReleaseSidebar = ({$c, release}: Props) => {
             artwork={releaseArtwork}
             fallback={releaseCoverUrl}
             message={ReactDOMServer.renderToStaticMarkup(exp.l(
-              'Front cover image failed to load correctly.<br/>{all|View all artwork}.',
+              `Front cover image failed to load correctly.
+               <br/>{all|View all artwork}.`,
```

Comment:
(For an unknown reason, the previous review is missing this comment.)

This is the only change that also changes localization message by adding a space before the `br` tag. Maybe it would be nice not to stop halfway, that is, to add a space after the `br` tag too:
```suggestion
               <br/> {all|View all artwork}.`,
```

Reply:
I would maybe use string concatenation here so we don't have to break the string to begin with